src/coreclr/vm/eetwain.cpp

```diff
@@ -2644,10 +2644,8 @@ unsigned scanArgRegTableI(PTR_CBYTE    table,
                 {
                     thisPtrReg = REGI_NA;
                 }
-                if  (iptrRegs & regMask)
-                {
-                    iptrRegs &= ~regMask;
-                }
```

Comment:
This change looks valid, but not directly related to other changes and the topic of this PR.